token/js/client/token.js

```diff
@@ -304,8 +313,15 @@ export class Token {
     publicKey: PublicKey,
     programId: PublicKey,
     payer: Account,
+    associatedProgramId: PublicKey,
```

Comment:
@jstarry I'm new to TS; but this is breaking api? Should I work-around?

Reply:
Yeah, work-around would be good to avoid the breaking api. I think that using the hard-coded associated program id is fine